src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Offender.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+class Offender(
+  private var referenceField: String,
+  private var nameField: String
+) {
+  val name get() = nameField
+  val reference get() = referenceField
```

Comment:
Does it need a class body at all? Will the name / reference fields ever be altered after an offender object is constructed? Just a question as I don't know how this might be used or evolve. It's fine for now though could simply be `class Offender (val reference: String = "", val name: String = "")` ?

Reply:
If I did that it created the Setters and I didn't want public setters (just public getters)